src/shared/ContentsTable/TableEntries/PullEntries/PullDirEntry.jsx

```diff
@@ -1,10 +1,15 @@
 import PropTypes from 'prop-types'
+import qs from 'qs'
+import { useLocation } from 'react-router-dom'
 
 import { usePrefetchPullDirEntry } from 'services/pathContents/pull/dir'
 
 import DirEntry from '../BaseEntries/DirEntry'
 
 function PullDirEntry({ pullId, urlPath, name, filters }) {
+  const { search } = useLocation()
+  const searchParams = qs.parse(search, { ignoreQueryPrefix: true })
+
```

Comment:
The filters here will include the search params, so you don't need to grab them from the URL